app/lib/airtable.server.ts

```diff
@@ -75,6 +75,10 @@ const scheduleResponseSchema = airtableResponseSchema(
       id: z.number(),
       Start: z.string(),
       Title: z.optional(z.string()),
+      Website: z.optional(stringArraySchema).default([]),
+      LinkedIn: z.optional(stringArraySchema).default([]),
+      Twitter: z.optional(stringArraySchema).default([]),
+      AvatarUrl: z.optional(stringArraySchema).default([]),
```

Comment:
Has the data model/API changed? When I hit it these are optional strings, not string arrays.

Reply:
As with previous note